migtests/tests/oracle-tests/ora-hr-db-sanity-test/validate

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+import yb
+
+def main():
+	yb.run_checks(migration_completed_checks)
+
+EXPECTED_ROW_COUNT = {
+	'countries': 25,
+	'departments': 27,
+	'employees': 107,
+	'jobs': 19,
+	'job_history': 10,
+	'locations': 23,
+	'regions': 4
+}
+
+QUERIES_CHECK = {
+	'FAULTY_QUERY_LESSER_END_DATE': {
+		'query': "INSERT INTO job_history (employee_id, start_date, end_date, job_id, department_id) VALUES (102, '2003-01-01', '2002-01-01', 'AD_PRES', 90);",
+		'code': 23514
+	}, 
+	'VIOLATE_INDEX_jhist_emp_id_st_date_pk': {
+		'query': "INSERT INTO job_history (employee_id, start_date, end_date, job_id, department_id) VALUES (102, '2001-01-13', '2006-07-24', 'AD_PRES', 90);",
+		'code': 23505
+	},
+	'VIOLATE_CHECK_CONSTRAINT_emp_salary_min': {
+		'query': "INSERT INTO employees (employee_id, first_name, last_name, email, phone_number, hire_date, job_id, salary, commission_pct, manager_id, department_id) VALUES (207, 'John', 'Doe', 'john.doe.d@dump.com', '515.123.8181', '2002-06-07', 'AC_ACCOUNT', 0, NULL, 205, 110);",
+		'code': 23514
+	}
+}
+
+#=============================================================================
+
+def migration_completed_checks(tgt):
+    # Logic to check for the total number of tables migrated
```

Comment:
Drop the words `Logic to ` from all of the comments. One can say, every statement is a "logic to achieve something".

Use complete statements in the comments with proper punctuation:
`Ensure that all the tables are migrated.`